c++/src/kj/mutex.h

```diff
@@ -162,7 +177,7 @@ class Once {
     virtual void run() = 0;
   };
 
-  void runOnce(Initializer& init);
+  void runOnce(Initializer& init, const LockSourceLocation& location = LockSourceLocation::current());
```

Comment:
It's a good idea but unfortunately the only place that can actually work is `runOnce`. At the other call-sites it comes after default variables & since there's no longer macros I can't move it to be a non-default parameter without breaking a lot of existing code. This feels like a change that would make sense if it could be applied in all 3 places, but just for `runOnce` this feels perhaps not worth the effort?

Reply:
> At the other call-sites it comes after default variables

But clearly those defaults are no longer relied upon either, so you might as well make them not defaulted?

Reply:
Ah, that's a good point I hadn't considered. Let me try that.